rrfs-test/IODA/python/bufr2ioda_msonet_prepbufr.py

```diff
@@ -0,0 +1,562 @@
+        .write_data(tob)
+
+    # ObsValue: Virtual Temperature
+    obsspace.create_var('ObsValue/virtualTemperature', dtype=tob.dtype, fillval=tob.fill_value) \
+        .write_attr('units', 'K') \
+        .write_attr('long_name', 'Virtual Temperature') \
+        .write_data(tvo)
+
+    # ObsValue: DewPoint Temperature
+    obsspace.create_var('ObsValue/dewPointTemperature', dtype=tob.dtype, fillval=tob.fill_value) \
+        .write_attr('units', 'K') \
+        .write_attr('long_name', 'DewPoint Temperature') \
+        .write_data(tdo)
+
+    # ObsValue: Specific Humidity
+    obsspace.create_var('ObsValue/specificHumidity', dtype=qob.dtype, fillval=qob.fill_value) \
+        .write_attr('units', 'kg kg-1') \
+        .write_attr('long_name', 'Specific Humidity') \
+        .write_data(qob)
+
+    # ObsValue: Eastward Wind
+    obsspace.create_var('ObsValue/windEastward', dtype=uob.dtype, fillval=uob.fill_value) \
+        .write_attr('units', 'm s-1') \
+        .write_attr('long_name', 'Eastward Wind') \
+        .write_data(uob)
+
+    # ObsValue: Northward Wind
+    obsspace.create_var('ObsValue/windNorthward', dtype=vob.dtype, fillval=vob.fill_value) \
+        .write_attr('units', 'm s-1') \
+        .write_attr('long_name', 'Northward Wind') \
+        .write_data(vob)
+
+    # Quality Marker: Pressure Quality Marker
+    obsspace.create_var('QualityMarker/pressure', dtype=pobqm.dtype, fillval=pobqm.fill_value) \
+        .write_attr('long_name', 'Pressure Quality Marker') \
+        .write_data(pobqm)
+
+    # Quality Marker: Station Pressure Quality Marker
+    obsspace.create_var('QualityMarker/stationPressure', dtype=pobqm.dtype, fillval=pobqm.fill_value) \
+        .write_attr('long_name', 'Station Pressure Quality Marker') \
+        .write_data(psqm)
+
+    # Quality Marker: Air Temperature Quality Marker
+    obsspace.create_var('QualityMarker/airTemperature', dtype=tobqm.dtype, fillval=tobqm.fill_value) \
+        .write_attr('long_name', 'Temperature Quality Marker') \
+        .write_data(tobqm)
+
+    # Quality Marker: Specific Humidity Quality Marker
+    obsspace.create_var('QualityMarker/specificHumidity', dtype=qobqm.dtype, fillval=qobqm.fill_value) \
+        .write_attr('long_name', 'Specific Humidity Quality Marker') \
+        .write_data(qobqm)
+
+    # Quality Marker: Eastward Wind Quality Marker
+    obsspace.create_var('QualityMarker/windEastward', dtype=uobqm.dtype, fillval=uobqm.fill_value) \
+        .write_attr('long_name', 'Eastward Wind Quality Marker') \
+        .write_data(uobqm)
+
+    # Quality Marker: Northward Wind Quality Marker
+    obsspace.create_var('QualityMarker/windNorthward', dtype=vobqm.dtype, fillval=vobqm.fill_value) \
+        .write_attr('long_name', 'Northward Wind Quality Marker') \
+        .write_data(vobqm)
+
+    # ObsError: Pressure Observation Error
+    obsspace.create_var('ObsError/pressure', dtype=poboe.dtype, fillval=poboe.fill_value) \
```

Comment:
I think we can just rename to stationPressure.

Reply:
@delippi done!